README.md

```diff
@@ -1,3 +1,5 @@
+:warning: **This repository is now archived as the feature is directly available when installing [`jupyter-collaboration`](https://github.com/jupyterlab/jupyter_collaboration)**
```

Comment:
Since the repo was renamed to use a `-` instead of `_`:

```suggestion
:warning: **This repository is now archived as the feature is directly available when installing [`jupyter-collaboration`](https://github.com/jupyterlab/jupyter-collaboration)**
```